FeedStoreChallenge/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,27 @@
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
+	@NSManaged internal var id: UUID
+	@NSManaged internal var imageDescription: String?
+	@NSManaged internal var location: String?
+	@NSManaged internal var url: URL
+	@NSManaged internal var cache: ManagedCache
```

Comment:
`cache` is optional in the `FeedStore.xcdatamodel`. Please make it optional here so that the model and the class are consistent
```suggestion
	@NSManaged var cache: ManagedCache?
```